sda/cmd/mapper/mapper.go

```diff
@@ -110,7 +110,7 @@ func main() {
 					}
 					err = inbox.RemoveFile(filePath)
 					if err != nil {
-						log.Errorf("Remove file from inbox failed, reason: %v", err)
+						log.Errorf("Remove file from inbox failed, corr-id: %s, reason: %v", delivered.CorrelationId, err)
```

Comment:
```suggestion
						log.Errorf("Remove file from inbox failed, reason: %v", err)
```
Mapping messages doesn't have any correlationID